tests/e2e/map/map_shapefile.spec.js

```diff
@@ -587,5 +657,219 @@ test.describe('Map: Shapefile interactions', () => {
+      })
+    })
+
+    test.describe('When the shapefile has a MultiPoint shape', () => {
+      test('renders correctly and filters collections @screenshot', async ({ page }) => {
+        await interceptUnauthenticatedCollections({
+          page,
+          body: commonBody,
+          headers: commonHeaders,
+          additionalRequests: [{
+            body: multipleShapesShapefileBody,
+            headers: {
+              ...commonHeaders,
+              'cmr-hits': '2'
+            },
+            paramCheck: (parsedQuery) => parsedQuery?.point?.[0] === '-109.6,38.81' && parsedQuery?.point?.[1] === '-109.55,38.75' && parsedQuery?.point?.[2] === '-109.5,38.7'
+          }]
+        })
+
+        await page.route(/shapefiles$/, async (route) => {
+          await route.fulfill({
+            json: { shapefile_id: '1' },
+            headers: { 'content-type': 'application/json; charset=utf-8' }
+          })
+        })
+
+        const initialMapPromise = page.waitForResponse(/World_Imagery\/MapServer\/tile\/2/)
+        await page.goto('/search')
+
+        // Wait for the map to load
+        await initialMapPromise
+
+        // Upload the shapefile
+        const shapefilePromise = page.waitForResponse(/World_Imagery\/MapServer\/tile\/11/)
+        await uploadShapefile(page, 'multipoint.geojson')
+        await shapefilePromise
+
+        // Populates the spatial display field
+        await expect(
+          page.getByTestId('filter-stack__spatial')
+            .locator('.filter-stack-item__secondary-title')
+        ).toHaveText('Shape File')
+
+        await expect(page.getByTestId('spatial-display_shapefile-name')).toHaveText('test.geojson')
+        await expect(page.getByTestId('filter-stack-item__hint')).toHaveText('1 shape selected')
+
+        // Checking that the right number of results are loaded ensures that the route
+        // was fulfilled correctly with the successful paramCheck
+        await expect(page.getByText('Showing 2 of 2 matching collections')).toBeVisible()
+
+        // Waiting for the URL to include the correct zoom level ensures the map is finished drawing - flexible zoom
+        await page.waitForURL(/zoom=11/, { timeout: 3000 })
+
+        // Draws the spatial on the map
+        await expect(page).toHaveScreenshot('multipoint.png', {
+          clip: screenshotClip,
+          maxDiffPixelRatio: 0.005
+        })
+
+        // Updates the URL and expect all points in the URL
+        await expect(page).toHaveURL(/search\?sp\[0\]=-109.6%2C38.81&sp\[1\]=-109.55%2C38.75&sp\[2\]=-109.5%2C38.7&sf=1&sfs\[0\]=0&lat=38\.\d+&long=-109\.\d+/)
+      })
+
+      test('renders selected multipoint correctly @screenshot', async ({ page }) => {
```

Comment:
This seems to be the same test as above. Did you mean to test the deselected multipoint? If so you should do the same beforeEach as above and pull out all the shared logic so you don't have to repeat code